website/content/partials/pki-forwarding-bug.mdx

```diff
@@ -0,0 +1,10 @@
+## PKI Certificate Generation Forwarding Regression
+
+A bug introduced in Vault 1.8 causes certificates issue by the PKI secrets engine made on a performance
```

Comment:
```suggestion
A bug introduced in Vault 1.8 causes certificates generation requests via the PKI secrets engine made on a performance
```

Reply:
I think "causes certificate generation requests via" (get rid of the pluralization of "certificates")?